python/paddle/fluid/tests/unittests/dygraph_to_static/test_list.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle.fluid as fluid
+from paddle.fluid.dygraph.jit import dygraph_to_static_graph
+
+SEED = 2020
+np.random.seed(SEED)
+
+
+def test_list_without_control_flow(x):
+    x = fluid.dygraph.to_variable(x)
+    a = []
+    a.append(x)
+    return a
+
```

Comment:
Will all `list.append()` will be transformed into `array_write`? I Unittest with plain `list.append()`  needed to add here?

Reply:
> Will all `list.append()` will be transformed into `array_write`?

Of course not.  This test ensures that python list will not be transformed.